### DISCUSSION TITLE
hey guys

### DISCUSSION THREAD

**Discussion by raheggs** [community/community#141607] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I just want to say hi to everyone

**Comment by Berchez** [comment]
Hi

**Comment by MingwieJin** [comment]
whats up man

**Comment by AFN4NX** [comment]
hello bro..

**Comment by Jakcrimson** [comment]
what's up man ?

**Comment by BrunoL0M** [comment]
hi

**Comment by ebndev** [comment]
Welcome to GitHub and the GitHub Community @raheggs 🎆! We're glad you're here and can't wait to see where your GitHub journey takes you.

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and connect with other users there! 👋
